libs/components/forms/src/lib/modules/character-counter/character-counter.directive.ts

```diff
@@ -70,24 +72,26 @@ export class SkyCharacterCounterInputDirective implements Validator {
         },
       };
     }
+
+    return {};
```

Comment:
Should we return `null` instead? Won't a non-null empty object be communicated as an error?

Reply:
Done